app/(aux)/circleSettings.js

```diff
@@ -343,7 +355,7 @@ export default function Page() {
 								iconSize={60}
 								icon='musical-notes'
 								iconColor='white'
-								href='/mainViewLayout'
+								href='/'
 							/>
 
 							<StyledText className='top-[20%] text-3xl text-offwhite'>
```

Comment:
Like with delete, this should reflect the current circle name

Reply:
[Done]

Reply:
[done]